### PR TITLE
Migrator Servlet Init

### PR DESCRIPTION
**Pull request by TomiwaOke**
Setting the basic file structure for the UI/Servlet work for the migration tool 
Work based off Maven AppEngine Skeleton Archetype

### CODE REVIEW

File: migrator/migrator-servlet/pom.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        <configuration>
+          <excludes>
+            <exclude>javax.servlet:javax.servlet-api</exclude>
+            <exclude>com.google.guava:guava</exclude> <!-- avoid android version -->
+          </excludes>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
```
Comment (brittanyroesch) [inline migrator/migrator-servlet/pom.xml]: nit: new line at the end

File: migrator/migrator-servlet/src/main/java/com/google/phonenumbers/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        + SystemProperty.version.get() + " Java " + properties.get("java.specification.version"));
+  }
+
+  public static String getInfo() {
+    return "Version: " + System.getProperty("java.version")
+        + " OS: " + System.getProperty("os.name")
+        + " User: " + System.getProperty("user.name");
+  }
+
+}
```
Comment (brittanyroesch) [inline migrator/migrator-servlet/src/main/java/com/google/phonenumbers/HelloWorld.java]: nit: new line

File: migrator/migrator-servlet/src/main/webapp/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <runtime>java8</runtime>
+  <threadsafe>true</threadsafe>
+  <system-properties>
+    <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
+  </system-properties>
+</appengine-web-app>
```
Comment (brittanyroesch) [inline migrator/migrator-servlet/src/main/webapp/WEB-INF/appengine-web.xml]: same here and the rest of the files :)

File: migrator/migrator-servlet/src/main/webapp/WEB-INF/logging.properties [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# A default java.util.logging configuration.
+# (All App Engine logging is through java.util.logging by default).
+#
+# To use this configuration, copy it into your application's WEB-INF
+# folder and add the following to your appengine-web.xml:
+#
+# <system-properties>
+#   <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
+# </system-properties>
+#
```
Comment (brittanyroesch) [inline migrator/migrator-servlet/src/main/webapp/WEB-INF/logging.properties]: You can remove all of the comments except for the first 2 lines

**Review by brittanyroesch — APPROVED** [review]

**Review by gonecoding — APPROVED** [review]

**Pull request merged**